packages/polling-controller/src/index.ts

```diff
@@ -8,4 +8,6 @@ export {
   StaticIntervalPollingController,
 } from './StaticIntervalPollingController';
 
+export { getKey } from './AbstractPollingController';
```

Comment:
It seems odd that we would need to expose this function, but see my comment above.